lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -90,7 +90,8 @@ def install!
                 unless skip_info_plist?(native_target)
                   create_info_plist_file(target.info_plist_path, native_target, target.version, target.platform, :additional_entries => target.info_plist_entries)
                 end
-                create_build_phase_to_symlink_header_folders(native_target)
+                target_file_accessor = target.file_accessors.detect { |fa| fa.spec.name == target.name }
```

Comment:
so this is meant to be always from the root spec?

Reply:
what happens if for a PodTarget only subspecs are enabled and not the root spec? Is this build phase completely missing?

Reply:
After thinking about it more, if any header_mapping_dirs are present, we'll need to have copy phases for all headers, right?

Reply:
yes I think thats right.